api_tests/nodes/views/test_node_list.py

```diff
@@ -1748,4 +1748,98 @@ def test_skip_uneditable_does_not_have_admin_permission_for_any_nodes(self):
         assert_equal(res.status_code, 403)
 
 
+class TestNodeListPagination(ApiTestCase):
 
+    def setUp(self):
+        super(TestNodeListPagination, self).setUp()
+        self.user_one = AuthUserFactory()
```

Comment:
You can speed up this method by using `UserFactory` rather than `AuthUserFactory`; `UserFactory` does not need to bcrypt passwords.